.travis.yml

```diff
@@ -17,6 +17,9 @@ jobs:
     - stage: Test
       name: AVA Regression Tests
       script: ava --tap -c 1 test/tests
+    - stage: Test Coverage
```

Comment:
Should the regression and lint jobs be done in separate stages too?

Reply:
hmmm actually I don't think this should be a separate stage. I thought perhaps it should be in a different category -- because it is a coverage report -- but I don't think the stages should be used like that. It seems the next stage doesn't run if the first one fails.